rusk-wallet/src/wallet.rs

```diff
@@ -398,7 +392,20 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
         derive_phoenix_vk(seed, index)
     }
 
-    /// Returns the shielded key for a given index.
+    /// get cache database path
+    pub(crate) fn cache_path(&self) -> Result<PathBuf, Error> {
+        let cache_dir = {
+            if let Some(file) = &self.file {
+                file.path().cache_dir()
+            } else {
+                return Err(Error::WalletFileMissing);
+            }
+        };
+
+        Ok(cache_dir)
+    }
+
+    /// Returns the Phoenix public-key for a given index.
```

Comment:
This was
```rust
/// Returns the shielded key for a given index.
```
before the changes. Please keep like that, we want to eliminate all references to phoenix or moonlight in docs.